pandas/tests/tslibs/test_period_asfreq.py

```diff
@@ -1,10 +1,32 @@
 # -*- coding: utf-8 -*-
 
+import pytest
+
+from pandas.errors import OutOfBoundsDatetime
 from pandas._libs.tslibs.frequencies import get_freq
```

Comment:
would not be averse to makeing this a submodule in future
tests/tslibls/period/test_asfreq